manifests/server.pp

```diff
@@ -25,23 +25,16 @@
 #   and remote connection activities. Anything from 127.0.0.1 will be dropped
 #   as useless.
 #
-# @param client_nets [NetList] A list of networks and/or hostnames that are
+# @param trusted_nets [NetList] A list of networks and/or hostnames that are
 #   allowed to connect to this service.
 #
 class rsync::server (
-  $use_stunnel = true,
-  $stunnel_port = '8730',
-  $listen_address = '0.0.0.0',
-  $drop_rsyslog_noise = true,
-  $client_nets = lookup('client_nets', Array, 'first', ['127.0.0.1'])
+  Boolean          $use_stunnel        = simplib::lookup('simp_options::stunnel', { default_value => true }),
```

Comment:
We've been using the catalyst name as the class parmeter for clarity, but it's not that big of a deal.